hnsecure/src/test/java/ca/bc/gov/hlth/hnsecure/parsing/FormatRTransResponseTest.java

```diff
@@ -0,0 +1,22 @@
+package ca.bc.gov.hlth.hnsecure.parsing;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import ca.bc.gov.hlth.hnsecure.test.TestMessages;
+
+/**
+ * @author anumeha.srivastava
+ *
+ */
+public class FormatRTransResponseTest{
+
+	@Test
+	public void testParseV2MessageInfoE45() throws Exception {
```

Comment:
For method naming it's always good to indicate the actual method being tested. E.g. testFormat(). This one looks like it was just copy/pasted.

If you want to be more specific about what data is being used to test, or what the expected outcome is then I like to add the additional information after an underscore. So your test method would be testFormat_R09() or something to that effect.